translations/tiled_ru.ts

```diff
@@ -398,17 +398,17 @@
     <message>
         <location line="+18"/>
         <source>No tilesets section found before layer section.</source>
-        <translation>Не найдено набора тайлов до слоя тайлов.</translation>
+        <translation>Перед разделом слоя не найдено раздела с набором тайлов.</translation>
     </message>
     <message>
         <location line="+28"/>
         <source>Error mapping tile id %1.</source>
-        <translation>Ошибка при создании карты для тайла с ID %1.</translation>
+        <translation>Ошибка при создании карты для тайла с id %1.</translation>
     </message>
     <message>
         <location line="+70"/>
         <source>This seems to be no valid flare map. A Flare map consists of at least a header section, a tileset section and one tile layer.</source>
-        <translation>Файл не соответствует спецификации Flare-карты. Flare карта состоит по крайней мере из одной секции заголовка, набора тайлов и слоя тайлов.</translation>
+        <translation>Файл не соответствует спецификации flare-карты. Flare карта состоит по крайней мере из одного раздела заголовка, раздела с набором тайлов и одного слоя тайлов.</translation>
```

Comment:
Why change "Flare" to lowercase?

Reply:
Sorry, that was my fault. I will redo the translation and will make new PR. You may decline this PR.

Reply:
I have done new pull request, where all strings were translated to Russian language. Please check.